kit/azure/activity-log/main.tf

```diff
@@ -1,3 +1,39 @@
+# Deploy "Configure Azure Activity logs to stream to specified Log Analytics workspace" Policy definition
+
+locals {
+
+  # policy Log Anaylytics Workspace
+  builtin_azurerm_policy_definition_names = {
+    activity_log = "2465583e-4e78-4c15-b6be-a36cbc7c8b0f"
+  }    
+  # policy set for vm and vm scaleset monitoring
+  builtin_azurerm_policy_set_definition_name_vm_mon = {
```

Comment:
i haven't looked up these policies, but they seem to me to implement

https://cloudfoundation.org/maturity-model/security-and-compliance/centralized-workload-and-infrastructure-logs.html

over https://cloudfoundation.org/maturity-model/security-and-compliance/centralized-audit-logs.html#how-implement-centralized-audit-logs

I'm not sure if it makes sense to separate these two things from one another. 
Centralized audit logs for Azure API access are absolute must have (stage 1 capability in CFMM) while workload and infrastructure logs such as monitoring VMs are probably more opinionated depending on what type of landing zone platform engineers are trying to build. 

E.g. if i build a serverless landing zone I might say I won't bother at all with deploying it and dealing with the fallout, e.g. https://www.azadvertizer.net/azpolicyinitiativesadvertizer/75714362-cae7-409e-9b99-a8e5075b7fad.html is pretty complex in its own right. Plus this one's already marked legacy again... Difficult to keep up with for small cloud foundation teams.